Makefile

```diff
@@ -106,13 +107,15 @@ lint: bin/golangci-lint ## Run linter
 	cd plugins/gateway/kong && ../../../bin/golangci-lint run
 	cd plugins/gateway/tyk/v3.2.2 && ../../../../bin/golangci-lint run
 	cd plugins/taper && ../../bin/golangci-lint run
+	$(MAKE) -C plugins/otel-collector lint
 
 .PHONY: fix
 fix: bin/golangci-lint ## Fix lint violations
 	cd backend && ../bin/golangci-lint run --fix
 	cd plugins/gateway/kong && ../../../bin/golangci-lint run --fix
 	cd plugins/gateway/tyk/v3.2.2 && ../../../../bin/golangci-lint run --fix
 	cd plugins/taper && ../../bin/golangci-lint run --fix
+	$(MAKE) -C plugins/otel-collector fix
```

Comment:
@jnapper7 can you please also add it under `license-check`?

Reply:
Done!